src/inventory_ui.h

```diff
@@ -129,7 +124,7 @@ class inventory_entry
         nc_color get_invlet_color() const;
         void update_cache();
 
-        inventory_entry_drawn_info drawn_info;
+        inclusive_rectangle drawn_rect;
```

Comment:
You're using inclusive rectangle, however the maximum x values you set for the rectangle are one-past the maximum x coordinates of the text. Either subtract one from the maximum x values or use half open rectangle and add one to the maximum y values.

Reply:
fixed